### PR TITLE
Address issues regarding the configuration.xml article

### PR DESCRIPTION
**Pull request by duncdrum**
addresses a number of open issues regarding the configuration.xml article

### CODE REVIEW

File: src/main/xar-resources/data/configuration.xml [hunk 1]
```diff
@@ -423,19 +415,54 @@
...
                 </varlistentry>
             </variablelist>
-            <para>The following sections describe the attributes and child elements of the above
-                elements.</para>
+            <para>The following sections describe the most commonly modified attributes and child elements of the above elements, including how to change the default behavior of exist-db's handling of whitespace characters.</para>
             <section>
                 <title>&lt;db-connection&gt;</title>
                 <para>This element contains basic default storage settings for eXist-db, including
-                    memory and system limits. Only one <sgmltag>db-connection</sgmltag> should be
+                    memory and system limits. Only one &lt;db-connection&gt; should be
```
Comment (joewiz) [inline src/main/xar-resources/data/configuration.xml]: Was there a problem with [`<sgmltag>`](http://tdg.docbook.org/tdg/4.5/sgmltag.html)? Seems like the semantically appropriate approach for inline tags in DocBook 4.5; DocBook 5 moved this to [`<tag>`](http://tdg.docbook.org/tdg/5.0/tag.html).

File: src/main/xar-resources/data/configuration.xml [hunk 2]
```diff
@@ -1350,12 +1405,12 @@ uri="http://exist-db.org/xquery/file" /&gt;</synopsis>
                     <synopsis language="xquery">import module namespace file="http://exist-db.org/xquery/file";</synopsis>
                     <para>Instead of providing a Java class, one can also specify a src URI which
                         must point to the XQuery source code of the module, e.g.:</para>
-                    <synopsis language="xml">&lt;module src="resource:org/exist/xquery/lib/json.xq" 
+                    <synopsis language="xml">&lt;module src="resource:org/exist/xquery/lib/json.xq"
```
Comment (joewiz) [inline src/main/xar-resources/data/configuration.xml]: Might I suggest replacing this example module (which I wouldn't suggest anyone use now that we have proper support for JSON in XQuery) with `<module uri="http://exist-db.org/xquery/kwic" src="resource:org/exist/xquery/lib/kwic.xql"/>`?

File: src/main/xar-resources/data/configuration.xml (hunk 1 above)
Comment (duncdrum) [reply src/main/xar-resources/data/configuration.xml]: well there is display problem with xml elements in section titles, which is not related to the docbook markup. My reason for switching was to increase consistency. Once we update the docbook markup for all contents, we might very well land on `<tag>` or `<sgmltag>`, but until then we should at least stick to one or the other on a per document basis

Comment (joewiz) [reply src/main/xar-resources/data/configuration.xml]: If we were consistent in other documents, I'd suggest we limit this change to titles—limiting our need for parsing string contents in a future pass. But if we use escaped angle brackets elsewhere for inline, then I guess we're inconsistent already and will need to do a pass on this anyway...

**Pull request merged**